code/game/objects/items/stacks/tape.dm

```diff
@@ -49,5 +52,16 @@
 			return
 	to_chat(user, span_info("You wrap [I] with [src]."))
 	use(1)
-	I.embedding = I.embedding.setRating(100, 10, 0, 0, 0, 0, 0, 0, TRUE)
+	I.embedding = I.embedding.setRating(100, fall_chance, 0, 0, 0, 0, removal_pain, removal_time, TRUE)
 	I.taped = TRUE
+
+/obj/item/stack/tape/guerrilla
+	name = "guerrilla tape"
+	singular_name = "guerrilla tape"
+	desc = "A suspicious looking roll of tape. It seems to be much more adhesive than the standard variety."
+	icon_state = "tape_evil"
+	amount = 5
+	removal_pain = 10
+	removal_time = 2 SECONDS //six seconds
```

Comment:
```suggestion
	removal_time = 2 SECONDS //six seconds
```

🤔

Reply:
yes it's six seconds, no i will not be taking questions at this time

Reply:
> yes it's six seconds, no i will not be taking questions at this time

????

Reply:
i have tested it and it is 6 seconds, i cannot tell you why